pkg/commands/loaders/worktrees.go

```diff
@@ -0,0 +1,55 @@
+package loaders
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/commands/models"
+	"github.com/jesseduffield/lazygit/pkg/commands/oscommands"
+	"github.com/jesseduffield/lazygit/pkg/common"
+	"path/filepath"
+	"strings"
+)
+
+type WorktreeLoader struct {
+	*common.Common
+	cmd oscommands.ICmdObjBuilder
+}
+
+func NewWorktreeLoader(
+	common *common.Common,
+	cmd oscommands.ICmdObjBuilder,
+) *WorktreeLoader {
+	return &WorktreeLoader{
+		Common: common,
+		cmd:    cmd,
+	}
+}
+
+func (self *WorktreeLoader) GetWorktrees() ([]*models.Worktree, error) {
+	worktreesOutput, err := self.cmd.New(`git worktree list --porcelain -z`).DontLog().RunWithOutput()
```

Comment:
<img width="1028" alt="image" src="https://user-images.githubusercontent.com/8456633/190872559-4f009248-c8b0-48af-852c-1e4af8626946.png">
I'm running git 2.33 and looks like that doesn't support the -z flag, so I'd remove that and use newlines instead (assuming that doesn't present any major problems).

Speaking of compatibility, we have a compatibility requirement with git v2.0+, and it's worth considering if any of this PR doesn't work with git v2.0. I know magit supports as early as 2.4, so I'm happy to bump up the minimum required version to 2.4 if necessary for anything in this PR.

Reply:
Based on 2 google searches, worktrees were introduced in [v2.5](https://github.blog/2015-07-29-git-2-5-including-multiple-worktrees-and-triangular-workflows/), so...

Reply:
shiiieeet haha

Reply:
In that case I reckon we just need to only run our worktree code if we're on at least 2.5, otherwise we hide the worktree tab